src/Reflection/ReflectionMethod.php

```diff
@@ -143,29 +129,19 @@ function ($docComment) {
             $docCommentList
         );
 
-        $docComment = '/**' . implode(PHP_EOL, $normalizedDocCommentList) . '*/';
-
-        return $docComment;
+        return '/**' . implode(PHP_EOL, $normalizedDocCommentList) . '*/';
     }
 
-    /**
-     * Fetch recursive doc blocks from parent classes
-     *
-     * @param \ReflectionClass $reflectionClass
-     * @param string           $methodName
-     *
-     * @return array|void
-     */
-    private function fetchRecursiveDocBlockFromParent($reflectionClass, $methodName)
+    private function fetchRecursiveDocBlockFromParent(ReflectionClass $reflectionClass, string $methodName): ?array
     {
         $docComment = [];
         $parentReflectionClass = $reflectionClass->getParentClass();
         if (! $parentReflectionClass) {
-            return;
+            return null;
```

Comment:
not to be changed in this PR, but do we want to keep null with arrays?
Usually it's easier to return empty array, so then we can iterate on the array etc.

Reply:
I like the suggested approach. Always returning arrays would make it easier.